R/datasets.R

```diff
@@ -9,3 +9,10 @@
 #'
 #' @source Racle et al. (2017), eLIFE, https://doi.org/10.7554/eLife.26476.029
 "dataset_racle"
+
+#' RNA-seq dataset with FACS measurements
+#'
+#' TODO
```

Comment:
@LorenzoMerotto could you please add some details about the dataset here?

Reply:
Did it!